src/test/java/requests/Triggers.java

```diff
@@ -0,0 +1,117 @@
+package requests;
+
+import configs.tests.PalauProperties;
+import healper.Constants;
+import healper.impl.BondImpl;
+import healper.impl.SmartContractImpl;
+import io.gatling.javaapi.core.ChainBuilder;
+import jp.co.soramitsu.iroha2.ExtensionsKt;
+import jp.co.soramitsu.iroha2.generated.AssetDefinitionId;
+import jp.co.soramitsu.iroha2.generated.SignedTransaction;
+import objects.Bond;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+
+import static healper.Constants.*;
+import static io.gatling.javaapi.core.CoreDsl.*;
+import static io.gatling.javaapi.http.HttpDsl.http;
+
+public class Triggers extends PalauProperties {
+
+    private static BondImpl bond = new BondImpl(new PalauProperties());
+
+    private static SmartContractImpl smartContract = new SmartContractImpl(new PalauProperties());
+
+    // it must be used onceOnly() controller
+    // DONE добавить в генезис в ассетдефинишн метадату /Users/michaeltimofeev/Documents/projects/laos-iroha2-adapter/docker-compose/palau/genesis.json
+    // наминтить ассет
+    // после стартавать смартконтракты
+
+    public static ChainBuilder registerBondAssetTrigger =
+            // it must use onlyOne() controller
+            exec(http("register bond_asset trigger")
+                    .post(session -> {
+                                return "/peer-0/" + Constants.URL_TRANSACTION;
+                            }
+                    )
+                    .body(ByteArrayBody(session -> {
+                                        return SignedTransaction.Companion.encode(smartContract.deployRegisterRegisterBondTrigger("019153fc-8d19-7faa-854a-57657ba0ed20"));
```

Comment:
What's it? Should it be a predefined constant?

Reply:
done